templates/zerver/help/unread-counter-and-pointer.md

```diff
@@ -0,0 +1,44 @@
+# How the unread counter and navigation work
+
+## Unread messages
+
+Zulip will consider a message read when the message is selected (the
+blue cursor box passes over it). Any message which is selected or
+above a message which is selected is marked as read. If you reach the
+very bottom of your feed, all the messages in view will be marked as
+read.
+
+## Navigation
+
+Zulip will always take you to the place you left off (your first
+unread message) to make it more easy for you to catch up with all the
+discussions that happened while you were away from your computer.
+
+Here is a more detailed overview of the navigation behavior:
+
+* When you click on a topic's name or the recipient list at the top of
+* a group of messages, Zulip will narrow you to that conversation and
```

Comment:
We only want a bullet at the start of this sentence, since with the line wrapping this is giving us a bullet for every chunk i.e.
* When you click on a topic's name or the recipient list at the top of
* a group of messages, Zulip will narrow you to that conversation and
* Zulip will select the message you were previously focused on in that
* conversation.